docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+---
+title: MUI X improvements, Charts in Alpha, Discord, and more!
+description: Support for time zones, a new filtering UI, charts and Discord
+date: 2023-07-30T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Date Pickers](#date-pickers)
+  - [Support for Time zones](#support-for-time-zones)
+  - [Digital Clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and Paste](#copy-and-paste)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+  - [Filter on Column headers](#filter-on-column-headers)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+  - [Performance Enhancements: Smoother Scrolling and Expanding Rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - Alpha version](#charts---alpha-version)
+- [Discord](#Discord)
+- [What's next](#whats-next)
+
+## Date pickers
+
+### Support for time zones
+
+No more hassle displaying dates on the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about the conversion logic and keeping the stored value consistent with your backend.
+
+<img src="/static/blog/mui-x-mid-v6-features/timezone.gif" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
```

Comment:
The gif looks pixelized, how about we use video instead? https://www.notion.so/mui-org/Tips-for-high-quality-media-5a1686758f4f4267989b67a1c2c270da?pvs=4#adeeea5189df47968e9c0aacebe87aec